stateoftheuniverse/widgets/constellations.py

```diff
@@ -0,0 +1,119 @@
+"""
+Get a list of constellations that will be visible from a location on the earth as a given time.
+"""
+#-------------------------
+#Imports
+#------------------------
+import warnings
+warnings.filterwarnings("ignore")
```

Comment:
Should probably take this out, we want to make sure the user gets to see any warnings that come through.